Bottom: 61520f42a7ee270566a49e6f39ab23257a9a49bf
Top:    948d7c849f13e8a747535c182e3def71bc2e2b92
Author: Jannis Leidel <jannis@leidel.info>
Date:   2018-02-28 21:14:30 +0100

Install redash-stmo.

In the long run we'll be able to install additional dependencies by
having an own Dockerfile to build images based on the Redash image
but that installs additional Python dependencies. But until we have
a fork with lots of changes ourselves we need to do it this way.

Redash-stmo contains the ability to hook up our own Dockerflow
library.

Refs #13
Refs #37


---
diff --git a/requirements.txt b/requirements.txt
index 97067862..60956ca8 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -61,3 +61,4 @@ disposable-email-domains
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
 gevent==1.4.0
+redash-stmo>=2018.4.0
